proxy-node-gateway/src/main/java/uk/gov/ida/notification/exceptions/mappers/ExceptionToErrorPageMapper.java

```diff
@@ -40,34 +36,9 @@ public Response toResponse(TException exception) {
 
     protected abstract Response.Status getResponseStatus(TException exception);
```

Comment:
https://github.com/alphagov/verify-proxy-node/blob/35f409399a3c4a204e3bce243d549920c215bd50/proxy-node-gateway/src/main/java/uk/gov/ida/notification/exceptions/mappers/ExceptionToErrorPageMapper.java#L37

This method and the overrides can be now removed as they're not used anywhere. Which will make the overrides of `ExceptionToErrorPageMapper` a bit useless but we'll deal with that later....